tag.go

```diff
@@ -158,6 +159,11 @@ func parseTag(fv reflect.Value, ft reflect.StructField) *Tag {
 	t.Xor = t.Get("xor")
 	t.Prefix = t.Get("prefix")
 	t.Embed = t.Has("embed")
+	negatable := t.Has("negatable")
+	if negatable && ft.Type.Name() != "bool" {
```

Comment:
You'll have to use `ft.Type.Kind() == reflect.Bool` here otherwise type aliases (eg. `type Bool bool`) will not work correctly.